faculty_cli/cli.py

```diff
@@ -710,24 +710,21 @@ def instance_types(verbose):
             click.echo(type_.name)
 
 
-@server.command(name="add-to-ssh-agent")
+@server.command(name="ssh-details")
```

Comment:
If I understand correctly, this is the default name that will be created from the function name, so this argument is redundant:

```suggestion
@server.command()
```